### PR TITLE
Add mobile navigation

### PR DESCRIPTION
**Pull request by tinydetoon**

### CODE REVIEW

File: web/css/mobile_menu.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0,0,0,0.6);
+    z-index: 2;
+    cursor: pointer;
+}
+
+.profile {
+    background:  url('../images/backgrounds/wallpaper.jpg') no-repeat;
```
Comment (Kern046) [inline web/css/mobile_menu.css]: Beware that line, I think the old wallpaper is too heavy to be loaded on mobile devices. It should be removed

File: app/Resources/views/base.mobile.html.twig [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                            <span>{{ 'user.account' | trans }}</span>
+                        </div>
+                    {% endif %}
+                         
+                 </div>
+                
+            </div>  
+
+            <div id="page-title"  >
+                   <button id="menu_button" onclick="showMenu()">Menu</button>
```
Comment (Kern046) [inline app/Resources/views/base.mobile.html.twig]: I think we can keep the desktop three bars button for menu toggling, mobile users are familiar with it

File: app/Resources/views/base.mobile.html.twig [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                    {% if is_granted('ROLE_USER') %} 
+                         <div class="menuelement" onclick='move("{{ path('dashboard') }}");'>
+                             <img src="{{ asset('images/icons/myspace.png') }}"/>
+                             <span> {{ 'user.dashboard' | trans }}</span>
+                         </div>
+                    {% endif %}
+                    {% if is_granted('ROLE_USER') %}
+                          <div class="menuelement" onclick='move("{{ path('homepage') }}");'>
+                              <img src="{{ asset('images/icons/notification.png') }}"/>
+                              <span> {{ 'notifcation.all' | trans }}</span>
```
Comment (Kern046) [inline app/Resources/views/base.mobile.html.twig]: Typo here :)

**Comment by Kern046** [comment]
I also think that the link "My Account" should be the user icon with the username. By the way, The username should be aligned with the icon, you can use flexboxes to do the trick ! Set the two elements in a ```display:flex``` div and then use the ```align-items``` property.

If we can't display the notifications right now, no problem. I'd like anyway that the user may see the number of unread notifications in the "All notifications" link :) !

Don't hesitate to ask for help if all the feedbacks are too much work to handle !

File: app/Resources/views/base.mobile.html.twig (hunk 3 above)
Comment (tinydetoon) [reply app/Resources/views/base.mobile.html.twig]: In fact is not really a typo error. I do not think we have created yet a view all notifications page. So I redirect to homepage for the moment ;-)

Comment (Kern046) [reply app/Resources/views/base.mobile.html.twig]: I was talking about "'notifcation.all'" ;)

**Pull request merged**